rdrf/rdrf/frontend/src/pages/proms_page/components/question.tsx

```diff
@@ -196,19 +223,32 @@ class Question extends React.Component<QuestionInterface, object> {
             return this.renderMultiSelect(question);
         }
 
+        let transformedInstruction: string[];
```

Comment:
could try "disjunctive type" expression: string[] | string

Reply:
The disjunctive type annotation has been used and it worked.